types/empower/tslint.json

```diff
@@ -1,11 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "dt-header": false,
-        "no-var-keyword": false,
-        "prefer-const": false,
-        "strict-export-declare-modifiers": false,
-        "typedef-whitespace": false,
-        "whitespace": false
+        "no-redundant-jsdoc-2": false,
```

Comment:
As above

Reply:
I don’t understand this rule, it seems contraproductive to me. I’m using `@param` and `@return` annotations for documentation, not types declaration. JSdoc documentation in types is very helpful and many type declarations contain it. So why?

Reply:
I don't believe there's any restriction on using JSDoc without types.  Are you getting errors when you re-enable the lint rule?  I removed the rules block locally (from both projects) and didn't see any errors.

Reply:
I’d swear that it failed on CI before, but it’s clear now.